content/shmem_request_wait.tex

```diff
@@ -16,21 +16,19 @@
 \apidescription{
     The \FUNC{shmem\_request\_wait} routine blocks until all outstanding operations on the request
     are finished. Outstanding operations are operations posted before
-    wait was called. After the routine returns, the request is set to
-    NULL. There are no guarantees on the ordering of the outstanding
+    wait was called.
+    There are no guarantees on the ordering of the outstanding
     operations.
- }
+}
 
 \apireturnvalues{
     None.
 }
 
 \apinotes{ In a threaded environment \FUNC{shmem\_request\_wait} from
 a thread other than the thread, posting the operation is permitted.
-
-!!! clarify
-However, care must be taken that no operations are posted while the
-request is being waited on.
+However, only one thread can call \FUNC{shmem\_request\_wait} for a
+particular request.
```

Comment:
The idea is right. However, we should make this more verbose. Probably, later ...